### DISCUSSION TITLE
Export data from the console

### DISCUSSION THREAD

**Discussion by charlyforot** [akvorado/akvorado#453] [Q&A]
Hello,

First of all, thank you for this great work!

I was wondering if it was planned to be able to make queries via the Akvorado console and add the fields you want and export the data?

Like Nfsen does for example (see picture in the link below)

[Nfsen-queries-console](https://www.vanimpe.eu/wp-content/uploads/2015/11/nfsen-cmd2-e1447192766758.png)

**Comment by vincentbernat** [comment]
This is not really planned actually. You can get the SQL query in the console and replay it directly with ClickHouse.
